app/models/log.rb

```diff
@@ -0,0 +1,61 @@
+class Log
+  require 'elasticsearch/persistence/model'
+  include Elasticsearch::Persistence::Model
+
+  # TODO: Always capitalize method before saving, etc
+
+  index_name "log_#{Rails.env}"
+
+  attribute :http_method, String
+  attribute :request_id, String
+  attribute :status, String
+  attribute :host, String
+  attribute :path, String, mapping: { analyzer: 'special_characters' }
+  attribute :fwd, String
+  attribute :raw, String
+  attribute :timestamp, DateTime
+
+  def to_s
+    if request_id.blank?
+      time = DateTime.parse raw
+      beautiful_time = time.strftime("%d/%m/%Y - %H:%M:%S %L")
+      ugly_date = time.strftime("%Y-%m-%dT%H:%M:%S.%6N%Z")
+      return "#{beautiful_time} #{raw.sub(ugly_date, '')}"
+    end
+
+    self.class.attributes_order.map(&:to_sym).map do |attribute|
+      value = self.send(attribute).to_s
+      if attribute == :created_at
```

Comment:
Will we ever need the **created_at** attribute not parsed? Can't we parse it **on_save** or something?